configcatclient/__init__.py

```diff
@@ -1,31 +1,40 @@
 from .configcatclient import ConfigCatClient
 from .interfaces import ConfigCatClientException
+from .datagovernance import DataGovernance
 
 
-def create_client(sdk_key):
+def create_client(sdk_key, data_governance=DataGovernance.Global):
     """
     Create an instance of ConfigCatClient and setup Auto Poll mode with default options
 
     :param sdk_key: ConfigCat SDK Key to access your configuration.
+    :param data_governance:
+    Set this parameter to restrict the location of your feature flag and setting data within the ConfigCat CDN.
+    Default: Global, This parameter must be in sync with the preferences on:
+    https://app.configcat.com/organization/data-governance (Only Organization Admins can set this preference.)
     """
-    return create_client_with_auto_poll(sdk_key)
+    return create_client_with_auto_poll(sdk_key, data_governance=data_governance)
 
 
 def create_client_with_auto_poll(sdk_key, poll_interval_seconds=60, max_init_wait_time_seconds=5,
                                  on_configuration_changed_callback=None, config_cache_class=None,
-                                 base_url=None, proxies=None, proxy_auth=None):
+                                 base_url=None, proxies=None, proxy_auth=None, data_governance=DataGovernance.Global):
     """
     Create an instance of ConfigCatClient and setup Auto Poll mode with custom options
 
     :param sdk_key: ConfigCat SDK Key to access your configuration.
     :param poll_interval_seconds: The client's poll interval in seconds. Default: 60 seconds.
     :param on_configuration_changed_callback: You can subscribe to configuration changes with this callback
-    :param max_init_wait_time_seconds: maximum waiting time for first configuration fetch in polling mode.
+    :param max_init_wait_time_seconds: maximum waiting time for first configuration fetch.
     :param config_cache_class: If you want to use custom caching instead of the client's default InMemoryConfigCache,
     You can provide an implementation of ConfigCache.
     :param base_url: You can set a base_url if you want to use a proxy server between your application and ConfigCat
     :param proxies: Proxy addresses. e.g. { 'https': 'your_proxy_ip:your_proxy_port' }
     :param proxy_auth: Proxy authentication. e.g. HTTPProxyAuth('username', 'password')
+    :param data_governance:
```

Comment:
Default: Global. Set this parameter to be in sync with the Data Governance preference on the Dashboard: https://app.configcat.com/organization/data-governance (Only Organization Admins have access)

Reply:
done